projects/go-lib/src/lib/components/go-radio/go-radio-group.component.html

```diff
@@ -1,11 +1,11 @@
 <fieldset
-  class="go-form__fieldset go-form__fieldset--no-margin"
   [disabled]="control.disabled"
-  [ngClass]="{ 'go-form__fieldset--error': control?.errors?.length, 'go-form__fieldset--dark': theme === 'dark' }"
+  [ngClass]="{ 'go-form__fieldset--error': control?.errors?.length, 'go-form__fieldset--dark': theme === 'dark', 'go-form__fieldset--disable': enableFieldset, 'go-form__fieldset go-form__fieldset--no-margin': enableFieldset === true }"
```

Comment:
I believe this can be simplified from:
```
'go-form__fieldset go-form__fieldset--no-margin': enableFieldset === true
```

to:
```
'go-form__fieldset go-form__fieldset--no-margin': enableFieldset
```